kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/subPages/PreviewSelectedResources.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+
+  <div>
+    <KCircularLoader v-if="loading" />
+    <PreviewContent
+      v-else
+      :currentContentNode="contentNode"
+      :ancestors="ancestors"
+      :isSelected="isSelected"
+      :questions="questions"
+      @addResource="handleAddResource"
+      @removeResource="handleRemoveResource"
+    />
+  </div>
+
+</template>
+
+
+<script>
+
+  import { getCurrentInstance } from 'vue';
+  import commonCoreStrings from 'kolibri/uiText/commonCoreStrings';
+  import useFetchContentNode from '../../../../../../composables/useFetchContentNode';
+  import useResourceSelection from '../../../../../../composables/useResourceSelection';
+  import { coachStrings } from '../../../../../common/commonCoachStrings';
+  import { PageNames } from '../../../../../../constants/index';
+  import PreviewContent from '../../PreviewContent';
+
+  export default {
+    name: 'PreviewSelectedResources',
+    components: {
+      PreviewContent,
+    },
+    mixins: [commonCoreStrings],
+    setup(props) {
+      const { contentNode, ancestors, questions, loading } = useFetchContentNode(props.contentId);
+      const { selectedResources, selectResources, deselectResources } = useResourceSelection();
+      const { manageLessonResourcesTitle$ } = coachStrings;
+      const instance = getCurrentInstance();
+
+      props.setTitle(manageLessonResourcesTitle$());
+      props.setGoBack(() => {
+        return instance.proxy.$router.go(-1);
```

Comment:
It seems that we dont want to have a back arrow here? At least the figma spec does not show it in this page:

![image](https://github.com/user-attachments/assets/d5fb9e48-8c7a-46bd-8c5f-2d424bdc3b01)

Reply:
@AlexVelezLl - good point! I'm not sure if this is intentional or not and I've been having a little bit of a chat with Jessica regarding navigation out of some of the leaves of the tree and how intuitive (or not) it is. I think for now, this doesn't need to be blocking, and as we test out the full work flow in a bug bash and Peter does manual QA as well, we will probably make some modifications (here and elsewhere) about the back button. because. it's always an adventure with back navigation. Just replying here because Allan asked me about the expected behavior in our checkin just now :)